lisk-dex-electron/src/BalanceDisplay.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import axios from 'axios';
 import { userContext } from './context';
-import formatThousands from './Utils'
+import {formatThousands} from './Utils'
```

Comment:
We should keep consistent spacing when destructuring objects.
Either change this to:

```
import { formatThousands } from './Utils'
```

Or remove the spaces everywhere else.